### PR TITLE
1h Swords Battania update to new tiers

### PR DESCRIPTION
**Pull request by AngryMob010**
All battania 1h swords given a thorough rebalance with new tier formula

Engraved Backsword
Decorated Broadsword
Highland Broad Blade
Broad Ridged Shortsword renamed to Fine Steel Broad Shortsword to fit in the lineup of broadswords better
Highland Fine Steel Blade renamed to Highland Highland Pointed Blade to indicate thrust specialty
Fine Steel Cavalry Broadsword
Broadsword renamed to Steel Broadsword to fit in the lineup of broadswords better
Ridged Iron Broadsword

### CODE REVIEW

File: crafting_pieces.xml [hunk 1]
```diff
@@ -3074,12 +3074,12 @@
       <Material id="Iron2" count="1" />
     </Materials>
   </CraftingPiece>
-  <CraftingPiece id="crpg_battania_noble_sword_2_t5_pommel" name="{=BKHKQ15N}Turqouise Pommel" tier="5" piece_type="Pommel" mesh="battania_pommel_8" culture="Culture.battania" length="2.5" weight="0.14">
+  <CraftingPiece id="crpg_battania_noble_sword_2_t5_pommel" name="{=BKHKQ15N}Turqouise Pommel" tier="5" piece_type="Pommel" mesh="battania_pommel_8" culture="Culture.battania" length="2.5" weight="0.001">
```
Comment (namidaka) [inline crafting_pieces.xml:3077]: 1 gram pommel?

File: crafting_pieces.xml [hunk 2]
```diff
@@ -3750,43 +3750,43 @@
...
   </CraftingPiece>
-  <CraftingPiece id="crpg_battania_sword_5_t5_handle" name="{=eBxzZ1SW}Jewelled Reinforced One Handed Grip" tier="5" piece_type="Handle" mesh="battania_grip_4" culture="Culture.battania" length="19.17" weight="0.225" item_holster_pos_shift="0,0,-0.03">
+  <CraftingPiece id="crpg_battania_sword_5_t5_handle" name="{=eBxzZ1SW}Jewelled Reinforced One Handed Grip" tier="5" piece_type="Handle" mesh="battania_grip_4" culture="Culture.battania" length="19.17" weight="0.15" item_holster_pos_shift="0,0,-0.03">
     <BuildData piece_offset="-1.88" previous_piece_offset="0.1" next_piece_offset="0.1" />
     <Materials>
       <Material id="Iron5" count="2" />
     </Materials>
   </CraftingPiece>
-  <CraftingPiece id="crpg_battania_noble_sword_2_t5_handle" name="{=SqR5K74b}Fine Highland Grip" tier="4" piece_type="Handle" mesh="battania_grip_1" culture="Culture.battania" length="15.5" weight="0.1">
+  <CraftingPiece id="crpg_battania_noble_sword_2_t5_handle" name="{=SqR5K74b}Fine Highland Grip" tier="4" piece_type="Handle" mesh="battania_grip_1" culture="Culture.battania" length="15.5" weight="0.001">
```
Comment (namidaka) [inline crafting_pieces.xml:3765]: 1 gram handle?

File: crafting_pieces.xml [hunk 3]
```diff
@@ -4107,21 +4107,21 @@
       <Material id="Iron3" count="1" />
     </Materials>
   </CraftingPiece>
-  <CraftingPiece id="crpg_battania_noble_sword_2_t5_guard" name="{=DLvSnz8U}Etched Highland Guard" tier="5" piece_type="Guard" mesh="battania_guard_5" culture="Culture.battania" length="5.7" weight="0.298">
+  <CraftingPiece id="crpg_battania_noble_sword_2_t5_guard" name="{=DLvSnz8U}Etched Highland Guard" tier="5" piece_type="Guard" mesh="battania_guard_5" culture="Culture.battania" length="5.7" weight="0.001">
```
Comment (namidaka) [inline crafting_pieces.xml:4110]: 1 gram guard?

**Review by namidaka — CHANGES_REQUESTED** [review]

**Comment by AngryMob010** [comment]
adjusted the 1 gram parts to 10 grams.

it is intentional for them to be very low values overall. was the only way to make the thrust speed high while keeping swing speed low.

**Review by namidaka — APPROVED** [review]

**Pull request merged**